platform/mellanox/docker-saiserver-mlnx.mk

```diff
@@ -22,12 +22,10 @@ $(DOCKER_SAISERVER_MLNX)_DEPENDS += $(SAISERVER) $(PYTHON_SDK_API)
 $(DOCKER_SAISERVER_MLNX)_PYTHON_DEBS += $(MLNX_SFPD)
 $(DOCKER_SAISERVER_MLNX)_LOAD_DOCKERS += $(DOCKER_CONFIG_ENGINE_STRETCH)
 SONIC_DOCKER_IMAGES += $(DOCKER_SAISERVER_MLNX)
-SONIC_STRETCH_DOCKERS += $(DOCKER_SAISERVER_MLNX)
```

Comment:
I am just curious why we had stretch target when we migrated to buster. Are you aware?

Reply:
Not sure, my impression is that saiserver is not in use so it probably didn't throw any errors.

Reply:
Got proven wrong by the RPC build here...